docs/instrumentations/php/native.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: 'Automatic Instrumentation'
+sidebarTitle: 'Native'
+icon: 'php'
+---
+
+<Info>
+  Native Instrumentation is currently the default setting of Odigos **Open-Source** version.
+
+  eBPF-Based Instrumentation is currently not supported for this language.
```

Comment:
suggestion: make it more positive and shorter:

```suggestion
  This language instrumentation is using OpenTelemetry native-php SDK and instrumentation libraries.
```

I think we can improve in other languages as well later on, but let's start with php 🎉